NitroxModel/Packets/BasePieceMetadataChanged.cs

```diff
@@ -7,11 +7,13 @@ namespace NitroxModel.Packets
     [Serializable]
     public class BasePieceMetadataChanged : Packet
     {
+        public NitroxId BaseParentId { get; }
         public NitroxId PieceId { get; }
         public BasePieceMetadata Metadata { get; }
 
-        public BasePieceMetadataChanged(NitroxId pieceId, BasePieceMetadata metadata)
+        public BasePieceMetadataChanged(NitroxId baseParentId, NitroxId pieceId, BasePieceMetadata metadata)
         {
+            BaseParentId = baseParentId;
```

Comment:
Is this really required? Can't you just use the parent or search for BasePiece where this id is needed?

Reply:
The client processor for basepiecemetadata still needs the piece id

Reply:
I mean the `baseParentId`, sry. Is this Id needed in many Metadata processor or only a few?

Reply:
baseParentId is necessary for the server's processor in the BaseManager